ddtrace/settings/config.py

```diff
@@ -225,6 +225,8 @@ def __init__(self):
 
         self._telemetry_metrics_enabled = asbool(os.getenv("_DD_TELEMETRY_METRICS_ENABLED", default=False))
 
+        self._128_bit_trace_id_enabled = asbool(os.getenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", False))
```

Comment:
For b3 and w3c propagation, we just continue to use the lower 64 bit trace ids correct? Just want to make sure.

Reply:
I called this out in the next steps. I plan to address this in my next PR. We will only propagate the 128bit trace ids for w3c and b3 if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=True`. Otherwise we'll continue to truncate it and use the lower 64 bits